dev_db/postgresql/image/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM postgres:9.6.6
```

Comment:
Maybe this should be 9.6 so we don't have to keep updating?

Reply:
As far as i know, 9.6.6 will work until we decide to switch to v10 because it is the latest (and last) version of the 9 series

Reply:
I agree that 9.7.x and 10.x should not be here because those have breaking changes. 

I'm proposing we use 9.6 not 9.6.6 so when we someone runs this script, it can automatically get 9.6.x because those should work and that's what we put in the dev docs.

Reply:
ok! :+1: